servicetalk-client-api/src/main/java/io/servicetalk/client/api/DefaultAutoRetryStrategyProvider.java

```diff
@@ -70,6 +76,19 @@ public Builder disableWaitForLoadBalancer() {
             return this;
         }
 
+        /**
+         * By default, automatic retries wait for the associated {@link LoadBalancer} to be ready or
```

Comment:
It would be useful to not cross-reference methods here as it is correct but more complex to understand. 

Simply, you can say that auto-retry strategies fail a request if the last signal from the associated `ServiceDiscoverer` was an error. This method disables that behavior.